basxconnect/mailer_integration/mailchimp/datasource.py

```diff
@@ -0,0 +1,37 @@
+from typing import List
+
+import mailchimp_marketing
+from django.conf import settings
+
+from basxconnect.mailer_integration.abstract import abstract_datasource
+from basxconnect.mailer_integration.abstract.abstract_mailer_person import MailerPerson
+from basxconnect.mailer_integration.mailchimp.mailchimp_person import MailchimpPerson
+
+URBANMOSAIC_ALL_MEMBERS_LIST_ID = "4606fb0179"
+SWISS_SEGMENT_ID = "36705"
+
+
+class MailchimpDatasource(abstract_datasource.Datasource):
+    def __init__(self) -> None:
+        self.client = mailchimp_marketing.Client()
+        self.client.set_config({"api_key": settings.MAILCHIMP_API_KEY, "server": "us5"})
+
+    def get_persons(self) -> List[MailerPerson]:
+        segment = self.client.lists.get_segment_members_list(
+            list_id=URBANMOSAIC_ALL_MEMBERS_LIST_ID,
+            segment_id=SWISS_SEGMENT_ID,
```

Comment:
This is also project specific.

Reply:
I will get rid of that one in the next PR (already working on it).

Reply:
actually, I cherry-picked it into that one...